sample-mfg/simulate-mfg.sh

```diff
@@ -37,7 +37,7 @@ if [[ "$1" == "-h" || "$1" == "--help" ]]; then
 fi
 : ${SDO_RV_URL:?}
 
-deviceBinaryDir='sdo_device_binaries_1.8_linux_x64'   # the place we will unpack sdo_device_binaries_1.8_linux_x64.tar.gz to
+deviceBinaryDir='sdo_device_binaries_1.9_linux_x64'   # the place we will unpack sdo_device_binaries_1.9_linux_x64.tar.gz to
```

Comment:
Would this change not affect people using SDO 1.8?

Reply:
No. See in section (Initialize a Test VM Device with SDO)[https://github.com/open-horizon/SDO-support/blob/master/README.md#initialize-a-test-vm-device-with-sdo] that it instructs the user to get `simulate-mfg.sh` from the branch https://github.com/open-horizon/SDO-support/releases/download/v1.8/simulate-mfg.sh